packages/backend/src/postHog.ts

```diff
@@ -47,12 +77,16 @@ export async function isFeatureFlagEnabled(
                       },
                   }
                 : {},
-        ),
-        new Promise<boolean>((resolve) => {
-            setTimeout(() => {
-                resolve(false);
-            }, FLAG_CHECK_TIMEOUT);
-        }),
+        );
+
+        clearTimeout(timeout);
```

Comment:
This will make it so that the `timeoutPromise` never rejects or resolves

Reply:
it does not matter because its in a `Promise.race`

Reply:
true, what I meant is that it'll hang in the background, but probably will be garbage collected anyway and it is not important 🤔

Reply:
🤞